src/main/resources/themes/one_dark.xml

```diff
@@ -0,0 +1,7 @@
+<scheme name="One Dark" parent_scheme="Darcula" version="142">
```

Comment:
We need this to be bundled. It is used when the user selects the theme in the `Appearance` settings. Then there's code that comes in after and puts the runtime generated scheme in.